### PR TITLE
ci: fix PyPI package deployment for pre-releases

### PR DESCRIPTION
**Pull request by redeboer**
Closes https://github.com/ComPWA/pycompwa/issues/51

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -126,10 +126,16 @@ jobs:
...
       env:
         - TASK="deployment"
-      install: echo "Skipping install stage!"
+        - ROOTBIN="v6.18.04.Linux-ubuntu18-x86_64-gcc7.4"
       before_install: echo "Skipping before_install stage!"
+      install: # install dependencies
+        - python3 -m pip install scikit-build
+        - wget https://root.cern.ch/download/root_${ROOTBIN}.tar.gz
+        - tar xpvfz root_*.tar.gz > /dev/null 2>&1
+        - source root/bin/thisroot.sh
```
Comment (spflueger) [inline .travis.yml:138]: do you really need this? I checked before and we only upload the source dist, so building should not be necessary right?

Comment (redeboer) [reply .travis.yml:138]: Yes I also saw that, neither did we install `scikit-build` previously, right? But then I don't understand why we receive [this error message](https://travis-ci.com/github/ComPWA/pycompwa/jobs/321134950#L998) now...

Comment (redeboer) [reply .travis.yml]: Okay removed that stuff again https://github.com/ComPWA/pycompwa/pull/87/commits/2798f46c358337002644f42da64586e398ab119c

Comment (redeboer) [reply .travis.yml]: Hope that renewing the PyPI key the actual fix
https://github.com/ComPWA/pycompwa/pull/87/commits/88391d1a56648d912a9a2fb7f524b7f830c35b57

File: .travis.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ env:
     - LANG: en_US.UTF-8
     - BASEPATH: "$TRAVIS_BUILD_DIR"
     - ROOTSYS: "$BASEPATH/root"
+    - secure: lHXkz6p4ZifmMwIAxSsCaIUsAl1iUYxbid4Q1CAdsCcyHty8UXUillrYH+jpk9yhk3aTZZezM4j1szDBsDdHAHi+lUT1rse+UFclFrID9MVcfNLEkL7CQTktQ6/IPt/vvCLEPGfi/u8qz2L2PcQ+73pyJfbA/GWMihU16c4yOwzz0mdzhs8UgiEQ4CyBqUGEROnnwK2M/4nfuIpPKwJMfSll3ebWiox2wZBSdpni0U7mN7IV63IrfPsjl0N2VBIFsog/528C8USIMqncL4ArotO+kSVZ1OKYgw/LZe7EtC+XrpQDJlQkgC6riQlDvS38sFSYE2ERawZ3AlfSIIMaDifi4Ag1G7N3gcMfjWHoecrZomyEtNKEdwZ+V1jH7DlNePl/pmSwU6i4Mvrw9a+MB+VfT81D2exax1aaC0aJMSNpPWCi5pf/X182yBwO/89oOYqtdTE7J5CaVzxBTjFk4pPPzDZgPyFPOyLMF8QtvtcnRP6DRe5p6Yt0Ztqc5KK9+J/xj8YrKVJoVPlIvVZfmkn5RdPL6f7fcnugb7vl52E0ig+qFjHMvQkQhahXl0gBWb6PYsEzUOcVXdVn917pcWBTjzw4y+x4qgygGsTYI9KxSJbM4cl3mqHsJOxgRdUzKegg2wB8TlMT+4fN1oOZOpi2Vp43Hb4EO59KOyMGGmY=
```
Comment (spflueger) [inline .travis.yml:17]: whats that?

Comment (redeboer) [reply .travis.yml:17]: See https://docs.travis-ci.com/user/deployment/pypi/ (https://github.com/ComPWA/pycompwa/pull/87/commits/88391d1a56648d912a9a2fb7f524b7f830c35b57)

Comment (redeboer) [reply .travis.yml:17]: Token has been renewed on PyPI

**Pull request merged**